lib/components/artwork_grids/artwork.js

```diff
@@ -30,6 +30,15 @@ class Artwork extends React.Component {
     )
   }
 
+  artist() {
+    const artist = this.props.artwork.artist
+    if (artist) {
+      return <SerifText style={[styles.text, styles.artist]}>{artist.name}</SerifText>
+    } else {
+      return null
```

Comment:
This would change the heights etc for any elements, making two not line up, is this fine? If so - this is good to go 👍